mitodl/ocw-studio
# Site detail page

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #40 

#### What's this PR do?
Implements the site detail page, and adds some scaffolding 

#### How should this be manually tested?
Find the name of a website you have permission to access in `/api/websites/`, then go to `/sites/website-name-here/`. You should see 2 or 3 links in the sidebar. You should be able to click on all of them and have the link change without a full page refresh. The "Content" page shouldn't have anything but the other links should go to component pages where you see a link at the top, similar to the mockup in the issue.
